user/views.py

```diff
@@ -18,7 +18,7 @@ class CreateTokenView(ObtainAuthToken):
 
 class ManageUserView(generics.RetrieveUpdateAPIView):
     serializer_class = UserSerializer
-    authentication_classes = (TokenAuthentication,)
+    # authentication_classes = (TokenAuthentication,)
```

Comment:
Consider uncommenting the `authentication_classes` if you intend to use token-based authentication. If using JWT, ensure the settings are correctly configured in the settings file.